clusterware-customize/libexec/customize/actions/help

```diff
@@ -152,6 +153,42 @@ help_for_trigger() {
 EOF
 }
 
+help_for_push() {
+  cat <<EOF
+  SYNOPSIS:
+
+    alces customize push
+
+  DESCRIPTION:
+
+    Push a customization profile to an upstream repository.
+EOF
+}
+
+help_for_slave() {
+  cat <<EOF
+  SYNOPSIS:
+
+    alces customize slave [OPTIONS]
+
+  DESCRIPTION:
+
+    Modifies the list of profiles to be ran when a compute node boots.
+    Has the option to add, remove or list the profiles.
+
+  OPTIONS:
```

Comment:
These are parameters (i.e. positional arguments) rather than options (i.e. `--flag`).

With this style of subcommand, it might be better to follow the pattern established by the `job-queue` action, which provides its own level of help.